products/views.py

```diff
@@ -67,15 +65,21 @@ def get(self, request):
         if colors:
             q &= Q(color__name__in = colors)
 
+        return q
+
+    def get(self, request): 
+
+        q = self.bulid(qs=request)
+
         products = ProductColor.objects.filter(q)
 
         result=[{
-            "primary_key": product.id,
-            "category"   : product.product.category.name,
-            "name"       : (product.product.name),
-            "color"      : product.color.name,
-            "price"      : product.product.price,
-            "image"      : [image.image_url for image in Image.objects.filter(product_color_id = product.id, sequence=1)]
+            "primary_key"  : product.id,
+            "category_name"     : product.product.category.name,
+            "product_name" : product.product.name,
+            "color_name"   : product.color.name,
+            "product_price": product.product.price,
```

Comment:
필드명을 조금 더 간단하게 만들어주셔도 충분히 파악이 가능합니다!
```suggestion
            "id": product.id,
            "category": product.product.category.name,
            "name": product.product.name,
            "color": product.color.name,
            "price": product.product.price,
```